src/Dfe.PlanTech.Web/ViewBuilders/PagesViewBuilder.cs

```diff
@@ -54,7 +56,8 @@ public async Task<IActionResult> RouteBasedOnOrganisationTypeAsync(Controller co
 
         if (page.IsLandingPage == true)
         {
-            return await BuildLandingPageAsync(controller, page);
+            var landingPageViewModel = await BuildLandingPageAsync(controller, page.Slug);
+            return controller.View(CategoryLandingPageView, landingPageViewModel);
         }
```

Comment:
I like this separation of concerns.

Minor non-blocking tweak: let's rename it to `BuildLandingPageViewModelAsync` perhaps?